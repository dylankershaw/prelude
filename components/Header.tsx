import React from 'react';

import {colors} from '../theme';

const Header = () => {
  return (
    <>
      <h1>Visualize your lifetime listening data.</h1>
      <div className='description'>
        To use this app, you'll first need a copy of your EndSong.json file from Spotify. You can
        obtain this by{' '}
        <a href='https://support.spotify.com/us/contact-spotify-support/'>
          contacting customer support
        </a>{' '}
        and requesting a copy of your lifetime listening data.
      </div>
      <style jsx>{`
        h1 {
          color: ${colors.green};
        }

        .description {
          margin-top: 1rem;
        }
      `}</style>
    </>
  );
};

export default Header;
