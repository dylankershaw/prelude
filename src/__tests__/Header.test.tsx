import React from 'react';
import {render, fireEvent, waitForElement} from '@testing-library/react';

import Header from '../components/Header';

test('matches snapshot', async () => {
  const {container} = render(<Header />);
  expect(container).toMatchSnapshot();
});
