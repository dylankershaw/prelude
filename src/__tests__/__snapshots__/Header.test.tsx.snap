// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <h1>
    Visualize your lifetime listening data.
  </h1>
  <div
    class="description"
  >
    To use this app, you'll first need a copy of your EndSong.json file from Spotify. You can obtain this by
     
    <a
      href="https://support.spotify.com/us/contact-spotify-support/"
    >
      contacting customer support
    </a>
     
    and requesting your lifetime listening data.
  </div>
  <style>
    
        h1 {
          color: #1DB954;
        }

        .description {
          margin-top: 1rem;
        }
      
  </style>
</div>
`;
